requirements.txt

```diff
@@ -11,6 +11,8 @@ python-telegram-bot==12.6.1
 Python-Twitter==3.5
 # Google
 simplifiedpytrends>=1.1.0
+# Ngrok
+pyngrok==2.1.4
```

Comment:
we should always freeze this version, this repo is very small, it could create a security issue with new versions